projects/noloan2/app/src/main/res/raw/itextkey.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<license date="2019-02-05" issuer="iText Group NV" library="3">
+    <licensee expire="2019-03-05">
```

Comment:
Expire on `2019-03-05`?

Reply:
Yes. This is the trial iText give for the "typography" module which fixes the Hebrew. 
https://itextpdf.com/en/resources/faq/legal-questions/itext-5/itext-java-library-free-charge-or-are-there-any-fees-be-paid

From what I understood - if used as open source & free app, legit to keep on using it. 
I can't say I'm a 100% sure for this. Still need to check, if you think it's worth. it.